packages/mongo-dev-server/README.md

```diff
@@ -0,0 +1,4 @@
+# mongo-dev-server
+[Source code of released version](https://github.com/meteor/meteor/tree/master/packages/mongo-dev-server) | [Source code of development version](https://github.com/meteor/meteor/tree/devel/packages/mongo-dev-server)
+***
+
```

Comment:
Would you mind adding one or two sentences describing the package in the README?